sonar-plugin-api/src/main/java/org/sonar/api/ce/posttask/Analysis.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package org.sonar.api.ce.posttask;
+
+import java.util.Date;
+
+public interface Analysis {
+  /**
+   * UUID of the analysis
+   */
+  String getAnalysisUuid();
+
+  /**
+   * Date of the analysis.
+   * <p>
+   * This date is the same as the date of the project analysis report and therefore as the analysis in DB. It can be
+   * missing when the status of the task is {@link org.sonar.api.ce.posttask.CeTask.Status#FAILED FAILED}.
```

Comment:
I think comment "It can be missing when the status of the task is {@link org.sonar.api.ce.posttask.CeTask.Status#FAILED FAILED}." now applies to the whole class, not the its Date field